### DISCUSSION TITLE
When I enter the commit message incorrectly

### DISCUSSION THREAD

**Discussion by x-satoshi** [community/community#136449] [Pull Requests]
### Select Topic Area

Question

### Body

How to come back?

**Comment by will2473** [comment]
How to put this free for  recovery phrase in exodus.. 
![Screenshot_2024-08-25-08-43-23-35_d6f621c6b7ed1d06b4ac8ef2db03391a](https://github.com/user-attachments/assets/e55ed532-e730-464d-b9c2-e642866b4405)
Help me for restore my wallet...!!!!
